libs/shared/ui/src/lib/utils/toast-error.spec.tsx

```diff
@@ -21,12 +21,42 @@ describe('error toaster', () => {
     }
 
     toastError(error)
-    expect(toast).toHaveBeenCalledWith(ToastEnum.ERROR, 'error', 'error message')
+    expect(toast).toHaveBeenCalledWith(
+      ToastEnum.ERROR,
+      'error',
+      'error message',
+      undefined,
```

Comment:
I'm not sure to understand, why you need to add several `undefined`?

Reply:
In this example, only the first param (`error`) is passed to the `toastError` function, meaning the underlaying `toast` function will be called with `undefined` values for the 6 other properties (i.e. `title`, `description`, `callback`, `iconAction`, `labelAction` and `externalLink`)

Reply:
Ok I thought you can by-pass adding`undefined` here with tests